BiometricLocker/BiometricLocker.swift

```diff
@@ -16,14 +16,33 @@ public protocol AuthenticationDelegate: class {
 }
 
 final public class BiometricLocker {
-    public enum Key: String {
+    /// Defines the behaviour of the `lock` function.
+    ///
+    /// - now: sets the app as locked immediatelly.
+    /// - afterTimeAllowance: sets the app as locked after the `unlockedTimeAllowace` interval at checking time.
+    /// - afterTimeInterval: sets the app as locked after the given time interval at checking time.
```

Comment:
Do these show up when you mouse over each of the enum cases someplace else you're calling them? If not, I recommend doing a `/// comment` above each case.